pages.ja/common/7za.md

```diff
@@ -1,25 +1,25 @@
 # 7za
 
-> 7-Zipは高圧縮率のファイルアーカイバ
-> より少ないアーカイブタイプをサポートする `7z`. スタンドアロンバージョン.
-> 詳しくはこちら: <https://www.7-zip.org>.
+> 圧縮率の高いファイルアーカイバ
+> `7z`よりも対応しているファイル形式は少ないですが、複数のプラットフォームに対応しています。
```

Comment:
Please put spaces between full-width characters and Latin characters.
```suggestion
> `7z` よりも対応しているファイル形式は少ないですが、複数のプラットフォームに対応しています。
```

Reply:
Thank you for the review! I've applied the suggestion.